bindata/etcd/pod.yaml

```diff
@@ -10,6 +10,35 @@ metadata:
     revision: "REVISION"
 spec:
   initContainers:
+    - name: defrag
+      image: ${IMAGE}
+      imagePullPolicy: IfNotPresent
+      terminationMessagePolicy: FallbackToLogsOnError
+      resources:
+        requests:
+          memory: 60Mi
+          cpu: 30m
+      volumeMounts:
+      - mountPath: /var/lib/etcd/
+        name: data-dir
+      env:
+${COMPUTED_ENV_VARS}
+      command:
+        - /bin/sh
+        - -c
+        - |
+          #!/bin/sh
+          set -euo pipefail
+
+          # Try and ensure etcd is really shut down.
+          echo -n "Waiting for ports 2379, 2380 and 9978 to be released."
+          while [ -n "$(lsof -ni :2379)$(lsof -ni :2380)$(lsof -ni :9978)" ]; do
```

Comment:
we have found out that `lsof` doesn't output ports in TIME_WAIT and migrated to `ss` https://github.com/openshift/cluster-kube-controller-manager-operator/blob/a45783f/bindata/v4.1.0/kube-controller-manager/pod.yaml#L21 (also added it to the base image)